docs/content/direct/architecture.md

```diff
@@ -102,11 +102,11 @@ in the [KubeFlex Architecture](https://github.com/kubestellar/kubeflex/blob/main
 There are currently two roles for spaces managed by KubeFlex: Inventory and Transport Space 
 (ITS) and Workload Description Space (WDS). The former runs the [OCM Cluster Manager](#ocm-cluster-manager) on a vcluster-type control plane, and the latter runs on a k8s-type control plane.
 
-An ITS holds the OCM inventory (`ManagedCluster`) objects and mailbox namespaces. The mailbox namespaces and their contents are implementation details that users do not deal with. Each mailbox namespace corresponds 1:1 with a WEC and holds `ManifestWork` objects managed by the central KubeStellar controllers.
+An ITS holds the inventory and the mailbox namespaces. The inventory is anchored by `ManagedCluster` objects that describe the WECs. For each WEC there may also be a `ConfigMap` object (in the `customization-properties` namespace) that carries additional properties of that WEC; this `ConfigMap` is used in cutomizing the workload to the WEC. The mailbox namespaces and their contents are transport implementation details that users do not deal with. Each mailbox namespace corresponds 1:1 with a WEC and holds `ManifestWork` objects managed by the central KubeStellar controllers.
 
```

Comment:
s/cutomizing/customizing/

Reply:
s/ users do not deal with/ users do not need to deal with/

Reply:
done